openmc/opencg_compatible.py

```diff
@@ -905,9 +905,12 @@ def get_opencg_lattice(openmc_lattice):
     for z in range(dimension[2]):
         for y in range(dimension[1]):
             for x in range(dimension[0]):
-                universe_id = universes[x][dimension[1]-y-1][z].id
+                universe_id = universes[z][dimension[1]-y-1][x].id
                 universe_array[z][y][x] = unique_universes[universe_id]
 
+    # Reverse y-dimension in array to match ordering in OpenCG
+    universe_array = universe_array[:, ::-1, :]
```

Comment:
Also the comment on line 911 should be changed to:

    # Reverse y-dimension in array to match ordering in OpenMC